creation/web_base/condor_startup.sh

```diff
@@ -216,18 +222,18 @@ set_var() {
         fi
     fi
 
-    if [ "$var_user" != "-" ]; then
+    if [[ "$var_user" != "-" ]]; then
         # - means do not export
-        if [ "$var_user" == "+" ]; then
+        if [[ "$var_user" == "+" ]]; then
             var_user=$var_name
-        elif [ "$var_user" == "@" ]; then
+        elif [[ "$var_user" == "@" ]]; then
             var_user=$var_condor
         fi
 
         condor_env_entry="$var_user=$var_val"
-        condor_env_entry=`echo "$condor_env_entry" | awk "{gsub(/\"/,\"\\\\\"\\\\\"\"); print}"`
-        condor_env_entry=`echo "$condor_env_entry" | awk "{gsub(/'/,\"''\"); print}"`
-        if [ -z "$job_env" ]; then
+        condor_env_entry=$(echo "$condor_env_entry" | awk "{gsub(/\"/,\"\\\\\"\\\\\"\"); print}")
```

Comment:
lines 234 and 235  fail after converting from backtick to $() notation.

[root@fermicloud528 glide_pvw10X]# condor_env_entry=`echo "$condor_env_entry" | awk "{gsub(/\"/,\"\\\\\"\\\\\"\"); print}"`
[root@fermicloud528 glide_pvw10X]# echo $condor_env_entry
""an oddly quoted string
[root@fermicloud528 glide_pvw10X]# condor_env_entry='"an oddly quoted string'
[root@fermicloud528 glide_pvw10X]# condor_env_entry=$(echo "$condor_env_entry" | awk "{gsub(/\"/,\"\\\\\"\\\\\"\"); print}")
awk: cmd. line:1: {gsub(/"/,"\\"\\""); print}
awk: cmd. line:1:               ^ backslash not last character on line
awk: cmd. line:1: {gsub(/"/,"\\"\\""); print}
awk: cmd. line:1:               ^ syntax error

Reply:
fixed using `condor_env_entry=$(echo "$condor_env_entry" | awk '{gsub(/"/,"\"\""); print}')` :
```
bash-3.2$ condor_env_entry='"an oddly quoted string'
bash-3.2$ condor_env_entry=$(echo "$condor_env_entry" | awk '{gsub(/"/,"\"\""); print}')
bash-3.2$ echo $condor_env_entry
""an oddly quoted string
```

Reply:
I double checked also the single quote one that was OK